.github/workflows/release.yml

```
@@ -13,6 +13,16 @@ jobs:
             - name: Checkout
               uses: actions/checkout@v3
 
+            - name: Backport
+              run: npm ci --ignore-scripts && npm run backport
```

I'd split this into 2 steps, to make it easier to read logs:
```suggestion
            - run: npm ci --ignore-scripts
            - run: npm run backport
```